frontend/src/stores/alert-store.ts

```diff
@@ -14,25 +14,24 @@ export const useSiteNotificationStore = defineStore('siteNotification', () => {
 
   /**
    * Check a given id if it is currently locked
-   * @param {string} checkId notification id to check agains current id
-   * @returns {boolean}
+   * @param checkId notification id to check agains current id
    */
-  const isSame = (checkId) => id.value === checkId;
+  const isSame = (checkId: string) => id.value === checkId;
 
   /**
    * Lock the given id
-   * @param {string} lockId notification id to lock in
```

Comment:
IMO it's distracting to hold the types in the doc comments too, which are properly defined the line below. But please let me know if you disagree on this.